packages/input-checkbox/src/Checkbox.js

```diff
@@ -40,32 +23,25 @@ const createCheckbox = ({ isSwitch }) =>
         disabled: inputProps.disabled,
         hasText: !!label
       };
-      const tooltipProps = {
-        onMouseLeave,
-        onMouseOver,
-        tooltipText,
-        tooltipAlign,
-        disabled: inputProps.disabled,
-        noTooltip
-      };
+
       inputProps.defaultChecked = checkedState;
       inputProps.onChange = this.onChange;
+
       const fontColor = inputProps.disabled
         ? fontTypes.SUBTLE
         : fontTypes.NORMAL;
+
       return (
         <StyledLabel {...labelProps}>
           <input type="checkbox" {...inputProps} className="hiddenCheckbox" />
-          <TooltipTarget {...tooltipProps} className="tooltipTarget">
-            <div className="visibleCheckbox">
-              <div className={isSwitch ? "toggle" : "checkmark wg-check"} />
-            </div>
-            {label && (
-              <Text title={!isSwitch} type={fontColor} size={fontSizes.MEDIUM}>
-                {label}
-              </Text>
-            )}
-          </TooltipTarget>
+          <div className="visibleCheckbox">
```

Comment:
Is this expected?
![2018-08-13 11 09 49](https://user-images.githubusercontent.com/17936244/44036963-8b12ba6c-9ee9-11e8-9e6b-b9aac81cd426.gif)

Reply:
It wasn't intentional, but this is a effect of removing the `tooltipTarget` div that became useless

Reply:
All the other components remained the same... weird

Reply:
Version 2.0.0:
![image](https://user-images.githubusercontent.com/17936244/44052097-1e0e782e-9f12-11e8-82d7-e62d0ddee18d.png)

After changes:
![image](https://user-images.githubusercontent.com/17936244/44052184-4c4bb5ee-9f12-11e8-8824-0582d0d9cdfc.png)

![2018-08-13 16 03 04](https://user-images.githubusercontent.com/17936244/44052229-68e8d5f6-9f12-11e8-9784-5eda52f9fa5a.gif)